build.gradle

```diff
@@ -65,14 +70,27 @@ spotless {
 
 
 // OpenAPI Server Generation
+// - Unzip external dependency on Cloud Resource Schema
 // - specify the source and target
 // - define the generation
 // - define the validation
 // - define dependencies to include the generated code
-
 def openapiSourceFile = "${projectDir}/src/main/resources/api/service_openapi.yaml"
 def openapiTargetDir = "${buildDir}/generated"
 
+configurations {
+    cloudResourceSchema { transitive = false }
+}
+dependencies {
+    cloudResourceSchema group: 'bio.terra.cloud-resource-lib', name: 'cloud-resource-schema', version: '0.0.0-SNAPSHOT'
```

Comment:
can we de-duplicate this (or at least the version) with what's on line 41?

Reply:
Maybe what we can do is 
(1) Declare the cloudResourceSchema configuration above. And merge those two dependencies.
(2) Let cloudResourceSchema's dependency externs implemeantion first
(3)Then we can have `cloudResourceSchema group: ...` thing once. 
But I feel that looks messy as we have things everywhere.  The current way is clean because we have one block for openAPI stuff and we can understand the steps dependencies. 
Feels like we didn't dedup anything and introducing extra work>

Reply:
version dedupe sgtm